build.gradle

```diff
@@ -65,6 +65,7 @@ ext {
     kotlinNativeUtilsModule="org.jetbrains.kotlin:kotlin-native-utils:${kotlinVersion}"
     kotlinUtilIoModule="org.jetbrains.kotlin:kotlin-util-io:${kotlinVersion}"
     kotlinUtilKlibModule="org.jetbrains.kotlin:kotlin-util-klib:${kotlinVersion}"
+    kotlinUtilKliMetadatabModule="org.jetbrains.kotlin:kotlin-util-klib-metadata:${kotlinVersion}"
```

Comment:
Why can't we have everything klib related in single JAR?

Reply:
+1, likely nobody needs KLIB reader w/o metadata

Reply:
util-klib-metadata has all the compiler internals such as descriptors as its dependency. 
util-klib has only ByteArray and String in its interfaces. 
A CLI tool to pack sources to the library would only need util-klib, but not util-klib-metadata.